### PR TITLE
(tech-89): access to zendesk

### PR DESCRIPTION
**Pull request by letbueno**
# Description
- Access to zendesk in Dapp App and personalizate 

# Necessary changes
- Add access to zendesk in config menu
- Add access to zendesk in giving page
- Personalizate zendesk

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] It is urgent
- [ ] <span style="color:#d44037">Dangerous change</span>

### CODE REVIEW

File: src/config/zendesk/index.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      `currentUser_id:${currentUser?.id}`,
+    ]);
+    ZendeskAPI("webWidget", "show");
+    ZendeskAPI("webWidget", "setLocale", currentLang);
+  };
+
+  return (
+    <ZendeskApp
+      defer
+      zendeskKey="efe9ba42-3eee-48a3-aa79-b573e297fef4"
```
Comment (thaisribon) [inline src/config/zendesk/index.tsx:39]: I think that key should be on env, right?

File: src/config/routes/index.tsx [hunk 2]
```diff
@@ -2,7 +2,6 @@ import React, { Suspense } from "react";
 import { Switch, Route } from "react-router-dom";
 import WalletProvider from "contexts/walletContext";
 import CausesPage from "pages/donations/CausesPage";
-import CurrentUserProvider from "contexts/currentUserContext";
```
Comment (thaisribon) [inline src/config/routes/index.tsx:5]: why was this removed?

File: src/config/zendesk/index.tsx (hunk 1 above)
Comment (letbueno) [reply src/config/zendesk/index.tsx:39]: yessss, i forgot thanksss

File: src/pages/promoters/SupportTreasurePage/CardSection/UserSupportSection/styles.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+export const Container = styled.div`
+  width: 100%;
+  max-width: 472px;
+  padding: 20px 14px 14px;
+  border-radius: 16px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  background-color: ${({ theme }) => theme.colors.orange20};
+  background-image: url(${cardBackground});
```
Comment (thaisribon) [inline src/pages/promoters/SupportTreasurePage/CardSection/UserSupportSection/styles.ts:14]: I think here we could use two images one for mobile and another for desktop, because the image on mobile does not look the same. If it's ok for Gu we can keep this way.

**Review by thaisribon — APPROVED** [review]
